### PR TITLE
add template support

### PR DESCRIPTION
**Pull request by iddar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,6 +28,6 @@ build/Release
 node_modules
 
 # Coffee genereated
-tasks
+#tasks
```
Comment (cybertk) [inline .gitignore]: tasks should be ignored?

File: tasks/raml2html.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+(function() {
```
Comment (cybertk) [inline tasks/raml2html.js]: This file should not be tracked as it's compiled by coffee

File: package.json [hunk 3]
```diff
@@ -39,5 +41,11 @@
   },
   "dependencies": {
     "raml2html": "^1.0.4"
-  }
+  },
```
Comment (cybertk) [inline package.json]: What does these following fields for?

File: .gitignore (hunk 1 above)
Comment (iddar) [inline .gitignore]: that was left after a test, sorry

File: package.json (hunk 3 above)
Comment (iddar) [inline package.json]: were inserted after testing for installing from the repository. Are not required

**Pull request closed without merging**